jina/resources/executors.requests.CompoundIndexer.yml

```diff
@@ -11,7 +11,7 @@ on:
       with:
         executor: BaseKVIndexer
         granularity_range: [0, 0]
-        adjacency_range: [0, 1]
+        adjacency_range: [1, 1]
```

Comment:
This is a change which I am not utterly sure of. But it somehow makes sense: once you retrieved the matches, you only collect the metadata for the matches and not of the query itself. BTW: Getting the metadata for the query itself broke the hello-world